### PR TITLE
Feature/run snyk

### PR DESCRIPTION
**Pull request by st3v3nhunt**

### CODE REVIEW

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder-pr-95 in nhsuk-dev succeeded (http://gp-finder-pr-95.dev.beta.nhschoices.net)

File: package.json [hunk 1]
```diff
@@ -4,18 +4,19 @@
...
-    "generate-coverage": "yarn run istanbul -- cover _mocha -- --recursive",
     "check-coverage": "istanbul check-coverage --config .istanbul.yml",
-    "upload-coverage-coveralls": "cat ./coverage/lcov.info | coveralls",
+    "generate-coverage": "yarn run istanbul -- cover _mocha -- --recursive",
+    "git-hook": "yarn run lint && yarn run generate-coverage && yarn run check-coverage && yarn run snyk -- test",
+    "istanbul": "NODE_ENV=test istanbul",
+    "lint": "eslint --ext .js,.json .",
+    "start": "node app.js",
+    "test": "NODE_ENV=test mocha --recursive test",
+    "travis": "yarn run git-hook && yarn run upload-coverage && yarn run snyk -- test --org=nhsuk && yarn run snyk -- monitor --org=nhsuk",
```
Comment (markysoft) [inline package.json]: Like it!

**Comment by st3v3nhunt** [comment]
@markysoft missing `bunyan` binary fixed

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder-pr-95 in nhsuk-dev succeeded (http://gp-finder-pr-95.dev.beta.nhschoices.net)

**Pull request merged**